### PR TITLE
Weld 2319

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

**Comment by tremes** [comment]
Feel free to suggest better `scanning information` UI formatting or displaying.

File: probe/core/src/main/client/probe.html [hunk 1]
```diff
@@ -380,6 +389,44 @@
                         </div>
                     </div>
                     {{/if}}
+                    <div class="form-group">
+                        <label class="col-sm-2 control-label">Scanning information:</label>
```
Comment (manovotn) [inline probe/core/src/main/client/probe.html]: Maybe we should only display this section if there is some content? Or have default message when there is nothing to display (`No scanning information available`)?

Comment (mkouba) [inline probe/core/src/main/client/probe.html]: I'm working on it ;-)

**Comment by mkouba** [comment]
@tremes @manovotn I've added few things... pls review.

**Comment by mkouba** [comment]
Merged, thanks!

**Pull request closed without merging**